src/values.md

```diff
@@ -0,0 +1,96 @@
+
+#### Values
+|Value   |Description   | Default
+|---|---|---|
+| isAppliance  | This indicates whether we are deploying tothe openmrs appliance or a cloud hosting   | true  |
```

Comment:
This indicates whether we are deploying **to the** openmrs appliance or a cloud hosting